### PR TITLE
Update DDF schema to version 2

### PR DESCRIPTION
**Pull request by skateman**
https://github.com/ManageIQ/manageiq-ui-classic/issues/7277

### CODE REVIEW

File: app/javascript/components/redfish-server-provision-dialog.jsx [hunk 1]
```diff
@@ -85,7 +79,8 @@ const RedfishServerProvisionDialog = ({ dispatch }) => {
       schema={schema}
       onSubmit={submitValues}
       showFormControls={false}
-      onStateUpdate={handleFormStateUpdate}
+      debug={handleFormStateUpdate}
```
Comment (himdel) [inline app/javascript/components/redfish-server-provision-dialog.jsx:82]: is this an old version before the formspy?

Comment (skateman) [reply app/javascript/components/redfish-server-provision-dialog.jsx:82]: No, if you go a few lines above, this is mapping a field value to the state. Therefore, it's not able to leverage the SpyField :disappointed: I'll fix it in a seaparate PR, so until then I'm using debug. Btw a few lines above there's some comment about it :wink:

Comment (himdel) [reply app/javascript/components/redfish-server-provision-dialog.jsx:82]: I've read the comment, still didn't know if it was up to date :) 

OK, works for me :+1:

**Review by himdel — APPROVED** [review]

File: app/javascript/components/redfish-server-provision-dialog.jsx (hunk 1 above)
Comment (skateman) [reply app/javascript/components/redfish-server-provision-dialog.jsx:82]: I have a feeling that we can get rid of this with a little rewrite which I'm planning after this is in :wink:

**Comment by skateman** [comment]
@miq-bot assign @Fryguy

**Review by agrare — APPROVED** [review]

**Pull request merged**